orc8r/tools/fab/pkg.py

```diff
@@ -15,13 +15,6 @@
 
 from fabric import Connection
 
-# Local changes are only allowed in files specified in the EXCLUDE_FILE_LIST
-EXCLUDE_FILE_LIST = [
-    os.path.realpath(x) for x in [
```

Comment:
There is now an unused import of `os` at the top of the file. Could you please remove it?